examples/nextjs/src/app/[[...slug]]/page.js

```diff
@@ -10,14 +11,14 @@ const client = dotcmsClient.init({
     }
 });
 
-import { MyPage } from '@/components/my-page';
-
 export async function generateMetadata({ params, searchParams }) {
     const data = await client.page.get({
         path: params?.slug ? params.slug.join('/') : 'index',
         language_id: searchParams.language_id,
         'com.dotmarketing.persona.id': searchParams['com.dotmarketing.persona.id'] || '',
-        mode: searchParams.mode
+        mode: searchParams.mode,
+        // This is the variant name that will be used to get the content from the experiments
+        variantName: searchParams['variantName'] ? searchParams['variantName'] : 'DEFAULT',
```

Comment:
We can set the `DEFAULT` in the `SDK`, right?

```suggestion
        variantName: searchParams?.['variantName'] ||  'DEFAULT',
```

Reply:
I changed this. with a comment of Jal.